### PR TITLE
AWS EKS CNI Mini tutorial

### PR DESCRIPTION
**Pull request by davidgs**
Please see the [contributing guidelines](CONTRIBUTING.md) for how to create and submit a high-quality PR for this repo. This template is based on Auth0's excellent template.

### Description

Super-short mini-tutorial for the AWS EKS CNI announcement.

@orishoshan 's blog will link to this

### CODE REVIEW

File: docs/quick-tutorials/aws-eks-cni-mini.mdx [hunk 1]
```diff
@@ -0,0 +1,161 @@
+---
+sidebar_position: 8
+title: AWS EKS CNI Mini-tutorial
+---
+import CodeBlock from "@theme/CodeBlock";
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Running the AWS VPC CNI plugin on EKS with Otterize. This mini-tutorial shows how to run the AWS VPC CNI plugin on EKS with Otterize.
```
Comment (orishoshan) [inline docs/quick-tutorials/aws-eks-cni-mini.mdx:9]:
```suggestion
This tutorial will walk you through deploying an AWS EKS cluster with the AWS VPC CNI add-on, while enabling the new network policy support on EKS with Otterize.
```

File: docs/quick-tutorials/aws-eks-cni-mini.mdx [hunk 2]
```diff
@@ -0,0 +1,161 @@
+---
+sidebar_position: 8
+title: AWS EKS CNI Mini-tutorial
```
Comment (orishoshan) [inline docs/quick-tutorials/aws-eks-cni-mini.mdx:3]:
```suggestion
title: Network policies on AWS EKS with the VPC CNI
```

File: docs/quick-tutorials/aws-eks-cni-mini.mdx [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+---
+import CodeBlock from "@theme/CodeBlock";
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Running the AWS VPC CNI plugin on EKS with Otterize. This mini-tutorial shows how to run the AWS VPC CNI plugin on EKS with Otterize.
+
+## Prerequisites
+
+* An EKS cluster with the AWS VPC CNI plugin installed. See [Installing the AWS VPC CNI plugin](https://docs.aws.amazon.com/eks/latest/userguide/pod-networking.html) for more information.
```
Comment (orishoshan) [inline docs/quick-tutorials/aws-eks-cni-mini.mdx:13]: This is the default really, so the key part here is to have network policies enabled.

Comment (orishoshan) [inline docs/quick-tutorials/aws-eks-cni-mini.mdx:13]:
```suggestion
* An EKS cluster with the AWS VPC CNI plugin installed and with the new built-in network policy support enabled. See [Installing the AWS VPC CNI plugin](https://docs.aws.amazon.com/eks/latest/userguide/pod-networking.html) for more information, or follow the instructions below.
```

File: docs/quick-tutorials/aws-eks-cni-mini.mdx [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+* The [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html).
+* The [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) command-line tool.
+
+## Step one: Create an AWS EKS cluster with the AWS VPC CNI plugin
+
+### Prepare a Kubernetes cluster
+
+Before you start, you'll need an AWS Kubernetes cluster. Having a cluster with a [CNI](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/) that supports [NetworkPolicies](https://kubernetes.io/docs/concepts/services-networking/network-policies/) is required for this tutorial.
+
+Save this `yaml` as `sample-config-file.yaml`:
```
Comment (orishoshan) [inline docs/quick-tutorials/aws-eks-cni-mini.mdx:25]: Maybe?
```suggestion
Save this `yaml` as `cluster-config.yaml`:
```

File: docs/quick-tutorials/aws-eks-cni-mini.mdx [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+  - name: vpc-cni
+    version: v1.12.6-eksbuild.2 # optional
+  - name: coredns
+    version: v1.10.1-eksbuild.1 # optional
+```
+
+Then run the following command to create your AWS cluster:
+
+```shell
+eksctl create cluster -f sample-config-file.yaml
```
Comment (orishoshan) [inline docs/quick-tutorials/aws-eks-cni-mini.mdx:59]:
```suggestion
eksctl create cluster -f cluster-config.yaml
```

File: docs/quick-tutorials/aws-eks-cni-mini.mdx [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+```
+
+Once everything is up and running from the AWS EKS side, so you can install the Otterize agents, and some pods to test the network policy creation.
```
Comment (orishoshan) [inline docs/quick-tutorials/aws-eks-cni-mini.mdx:62]:
```suggestion
Once your AWS EKS has finished deploying the control pane and node group, you can continue to deploy Otterize, and a couple of clients and a server to see how they are affected by network policies.
```

File: docs/quick-tutorials/aws-eks-cni-mini.mdx [hunk 7]
```diff
@@ -0,0 +1,161 @@
...
+
+## Step two: Install the Otterize agents
+
+### Install the Otterize agent on your cluster
```
Comment (orishoshan) [inline docs/quick-tutorials/aws-eks-cni-mini.mdx:66]:
```suggestion
### Deploy Otterize on your cluster
```

File: docs/quick-tutorials/aws-eks-cni-mini.mdx [hunk 8]
```diff
@@ -0,0 +1,161 @@
...
+Finally, you'll need to install the Otterize CLI (if you haven't already) to interact with the network mapper:
+
+<details>
+<summary>Install the Otterize CLI</summary>
+
+{@include: ../_common/install-otterize-cli.md}
+
+</details>
+
+### Install some basic pods
```
Comment (orishoshan) [inline docs/quick-tutorials/aws-eks-cni-mini.mdx:103]:
```suggestion
### Deploy a server and two clients
```

File: docs/quick-tutorials/aws-eks-cni-mini.mdx [hunk 9]
```diff
@@ -0,0 +1,161 @@
...
+kubectl apply -f https://docs.otterize.com/code-examples/automate-network-policies/all.yaml
+```
+
+Once you have that installed and running your Otterize access graph should look something like this:
+
+![Access Graph](/img/quick-tutorials/aws-eks-mini/access-graph.png)
+
+## Step three: Create an intent
+
+Now that you have the Otterize agents installed, you can create an intent to make sure that, when you apply security, your clients aren't blocked to the `server`.
```
Comment (orishoshan) [inline docs/quick-tutorials/aws-eks-cni-mini.mdx:117]:
```suggestion
Now that you have Otterize installed, the next step is to create an intent which will enable access to the server from the client. If you enable protection on the server without declaring an intent, the client will be blocked.
```

File: docs/quick-tutorials/aws-eks-cni-mini.mdx [hunk 10]
```diff
@@ -0,0 +1,161 @@
...
+
+```shell
+otterize network-mapper export --server server.otterize-tutorial-npol | kubectl apply -f -
+```
+
+At which point you should see that the `frontend` service is ready to be protected:
```
Comment (orishoshan) [inline docs/quick-tutorials/aws-eks-cni-mini.mdx:123]:
```suggestion
At which point you should see that the `server` service is ready to be protected:
```

File: docs/quick-tutorials/aws-eks-cni-mini.mdx [hunk 11]
```diff
@@ -0,0 +1,161 @@
...
+
+![One intent applied](/img/quick-tutorials/aws-eks-mini/one-intent.png)
+
+And you can then protect the `frontend` service by applying the following `yaml` file:
```
Comment (orishoshan) [inline docs/quick-tutorials/aws-eks-cni-mini.mdx:127]:
```suggestion
And you can then protect the `server` service by applying the following `yaml` file:
```

File: docs/quick-tutorials/aws-eks-cni-mini.mdx [hunk 12]
```diff
@@ -0,0 +1,161 @@
...
+apiVersion: k8s.otterize.com/v1alpha2
+kind: ProtectedService
+metadata:
+  name: server
+  namespace: otterize-tutorial-npol
+
+spec:
+  name: server
+```
+Save that to a file called `protect-server.yaml` and then run:
```
Comment (orishoshan) [inline docs/quick-tutorials/aws-eks-cni-mini.mdx:139]: Let's skip this step and just give them a URL they can access - by putting it in the code-examples dir.

File: docs/quick-tutorials/aws-eks-cni-mini.mdx (hunk 3 above)
Comment (davidgs) [reply docs/quick-tutorials/aws-eks-cni-mini.mdx:13]: Do I need to change anything?

File: docs/quick-tutorials/aws-eks-cni-mini.mdx [hunk 13]
```diff
@@ -0,0 +1,161 @@
...
+---
+import CodeBlock from "@theme/CodeBlock";
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This tutorial will walk you through deploying an AWS EKS cluster with the AWS VPC CNI add-on, while enabling the new network policy support on EKS with Otterize.
+
+## Prerequisites
+
+* An EKS cluster with the AWS VPC CNI plugin installed and with the new built-in network policy support enabled. See [Installing the AWS VPC CNI plugin](https://docs.aws.amazon.com/eks/latest/userguide/pod-networking.html) for more information, or follow the instructions below.
```
Comment (orishoshan) [inline docs/quick-tutorials/aws-eks-cni-mini.mdx:13]:
```suggestion
* An EKS cluster with the AWS VPC CNI add-on installed and with the new built-in network policy support enabled. See [Installing the AWS VPC CNI add-on](https://docs.aws.amazon.com/eks/latest/userguide/pod-networking.html) for more information, or follow the instructions below.
```

File: docs/quick-tutorials/aws-eks-cni-mini.mdx (hunk 12 above)
Comment (davidgs) [reply docs/quick-tutorials/aws-eks-cni-mini.mdx:134]: Done.

**Review by orishoshan — APPROVED** [review]

**Pull request merged**